tests/wast.rs

```diff
@@ -203,15 +203,8 @@ fn should_fail(test: &Path, strategy: Strategy) -> bool {
             }
         }
     }
-    let unsupported_gc_tests = [
-        "binary_gc.wast",
```

Comment:
Looks like some of these were because there weren't any actual files of that name, so must have been from some historical version of the gc proposal's spec tests or something like that?

Reply:
I think maybe some didn't have any assertions either? I don't know